src/System.Net.Http.Json/src/Configurations.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard;
```

Comment:
should we also have a net461 configuration?

Reply:
We could add a net461 config, but that would still mean we will need the facades here, since we depend on SYstem.Text.Json which will pull them in. I was planning on adding net461 config here with my configuration changes wave comming next so that we have one PR doing it for every package that needs it. If you still think I should add it here now, I can do that too.

Reply:
Go ahead and do it now.  I think the timing of the next wave will land before we stabalize this, so I'd prefer to have this done WRT coding and just need to pick up package dependency updates when we ship those.

Reply:
(assuming this is a small task, if anything gets complicated we can postpone this until after the first preview)